Tests/IntegrationTests/NavigatorConsistency.cpp

```diff
@@ -300,45 +300,46 @@ void runConsistencyTest(const propagator_probe_t& propProbe,
                                 refSurfaces.begin(), refSurfaces.end());
 }
 
+Acts::Logging::Level logLevel = Acts::Logging::INFO;
```

Comment:
This is not used everywhere. I'd probably go with a global `Logger` instance and clone it as needed.

Reply:
It is used a bit further down
```c++
  ACTS_LOCAL_LOGGER(Acts::getDefaultLogger("NavigatorTest", logLevel))
```